prime-router/metadata/schemas/upload-covid-19.schema

```diff
@@ -291,13 +291,13 @@ elements:
       mapperOverridesValue: true
       mapper: ifNotPresent($mode:lookup, testing_lab_street, ordering_facility_name_temp, ordering_facility_street_temp)
 
-    - name: ordering_facility_street2
-      csvFields: [{name: ordering_facility_street2}]
-
 #    - name: ordering_facility_street2
 #      csvFields: [{name: ordering_facility_street2}]
-#      mapperOverridesValue: true
-#      mapper: ifNotPresent($mode:lookup, testing_lab_street2, ordering_facility_name_temp, ordering_facility_street_temp)
+
+    - name: ordering_facility_street2
```

Comment:
Does the team really want these changes live?  Do not merge without having 100% confirmation

Reply:
I will confirm with @rhood23699 and make the team want these changes live.

Reply:
@rhood23699 told me to revert schema change back.  He will change it.